typings.json

```diff
@@ -3,5 +3,8 @@
     "core-js": "registry:dt/core-js#0.0.0+20160725163759",
     "jasmine": "registry:dt/jasmine#2.2.0+20160621224255",
     "node": "registry:dt/node#6.0.0+20160831021119"
+  },
+  "dependencies": {
+    "debug": "registry:npm/debug#2.0.0+20160723033700"
```

Comment:
If this came from a commit by Michael, shouldn't it be in the rebase? If this is here, we may have something else missing that would be conflict with master. We should check the reflog.